misc/python/materialize/parallel_workload/database.py

```diff
@@ -860,3 +863,28 @@ def create(self, exe: Executor) -> None:
 
         for relation in self:
             relation.create(exe)
+
+        result = composition.run(
```

Comment:
How much time do you think this action is going to take? Does it make sense, in order to obtain fair scheduling, to limit it to e.g. 1 second or 10 seconds?

Reply:
The run here takes a few seconds, but it only runs once in the beginning to get the catalog state from Materialize.

The interesting part is the SQLsmithAction, which takes less than a second. All it does is get the queries back from SQLsmith. Most of the time was previously spent in getting the catalog state from the database, we only do that once. Only the tables have to be refetched, which is pretty fast. If it turns out to be too slow, can doo